HHS/Head-Start-TTADP
# [TTAHUB-1326] Update topic/frequency graph with new design

## Description of change

Update the frequency of topics by AR widget with new design. The regional dashboard version of the graph also now includes the "take a screenshot" button.

## How to test

Ensure the graph still displays the correct information, the sorting works where appropriate, and the table view still displays what you'd expect. Test the responsive behavior and the screenshot button.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1326


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
